### ISSUE TITLE
Compatibility with SLEAP

### ISSUE THREAD

**Issue by IsabelleSajonia** [shlizee/OpenLabCluster#3]
Hi. Our lab is using SLEAP to get pose estimation results. Can OpenLabCluster work with a SLEAP H5 file or slp file?

**Comment by christincha** [comment]
Yes, you can use the SLEAP H5 file. As long as your file is organized correctly, each dataset in the H5 file is a sample that has data array T (=number of samples)* F (=number of key points * dimension).
